openhtf/plugs/__init__.py

```diff
@@ -482,7 +482,11 @@ def initialize_plugs(self, plug_types=None):
           # Now the instance has its own copy of the test logger.
           plug_instance.logger = self._logger
       except Exception:  # pylint: disable=broad-except
-        _LOG.error('Exception instantiating plug type %s', plug_type)
+        _LOG.error('Exception instantiating plug type %s', plug_type,
```

Comment:
Will do. I didn't realize this existed, but will start using it now that I know!